admin/resources/js/components/poolCandidate/messages.ts

```diff
@@ -0,0 +1,186 @@
+import { defineMessages } from "react-intl";
+
+export default defineMessages({
+  createHeading: {
+    id: "poolCandidate.form.createHeading",
+    defaultMessage: "Create Pool Candidate",
+    description: "Title displayed on the Update a User form.",
+  },
+  updateHeading: {
+    id: "poolCandidate.form.updateHeading",
+    defaultMessage: "Update Pool Candidate",
+    description: "Title displayed on the Update a User form.",
+  },
+  poolLabel: {
+    id: "poolCandidate.form.poolLabel",
+    defaultMessage: "Pool: ",
+    description: "Label displayed on the Pool Candidate form Pool field.",
+  },
+  poolPlaceholder: {
+    id: "poolCandidate.form.poolPlaceholder",
+    defaultMessage: "Select a pool...",
+    description: "Placeholder displayed on the Pool Candidate form Pool field.",
+  },
+  userLabel: {
+    id: "poolCandidate.form.userLabel",
+    defaultMessage: "User: ",
+    description: "Label displayed on the Pool Candidate form User field.",
+  },
+  userPlaceholder: {
+    id: "poolCandidate.form.userPlaceholder",
+    defaultMessage: "Select a user...",
+    description: "Placeholder displayed on the Pool Candidate form User field.",
+  },
+  cmoIdentifierLabel: {
+    id: "poolCandidate.form.cmoIdentifierLabel",
+    defaultMessage: "Cmo Identifier: ",
+    description:
+      "Label displayed on the Pool Candidate form Cmo Identifier field.",
+  },
+  expiryDateLabel: {
+    id: "poolCandidate.form.expiryDateLabel",
+    defaultMessage: "Expiry Date: ",
+    description:
+      "Label displayed on the Pool Candidate form Expiry Date field.",
+  },
+  isWomanLabel: {
+    id: "poolCandidate.form.isWomanLabel",
+    defaultMessage: "Woman: ",
+    description: "Label displayed on the Pool Candidate form Woman field.",
+  },
+  hasDisabilityLabel: {
+    id: "poolCandidate.form.hasDisabilityLabel",
+    defaultMessage: "Has Disability: ",
+    description:
+      "Label displayed on the Pool Candidate form Has Disability field.",
+  },
+  isIndigenousLabel: {
+    id: "poolCandidate.form.isIndigenousLabel",
+    defaultMessage: "Indigenous: ",
+    description:
+      "Placeholder displayed on the Pool Candidate form Indigenous field.",
+  },
+  isVisibleMinorityLabel: {
+    id: "poolCandidate.form.isVisibleMinorityLabel",
+    defaultMessage: "Visible Minority: ",
+    description:
+      "Label displayed on the Pool Candidate form Visible Minority field.",
+  },
+  hasDiplomaLabel: {
+    id: "poolCandidate.form.hasDiplomaLabel",
+    defaultMessage: "Has Diploma: ",
+    description:
+      "Label displayed on the Pool Candidate form Has Diploma field.",
+  },
+  languageAbilityLabel: {
+    id: "poolCandidate.form.languageAbilityLabel",
+    defaultMessage: "Language Ability: ",
+    description:
+      "Label displayed on the Pool Candidate form Language Ability field.",
+  },
+  languageAbilityPlaceholder: {
+    id: "poolCandidate.form.languageAbilityPlaceholder",
+    defaultMessage: "Select a language...",
```

Comment:
Should this be _Select a language ability_ since **Bilingual** is not a language?